sdk/eventhub/event-hubs/src/partitionProcessor.ts

```diff
@@ -0,0 +1,65 @@
+import { PartitionContext } from "./partitionContext";
+import { CheckpointManager } from "./checkpointManager";
+import { CloseReason } from "./eventProcessor";
+import { ReceivedEventData } from "./eventData";
+
+/**
+ * The `PartitionProcessor` is responsible for processing events received from Event Hubs when using `EventProcessor`
+ *
+ * An instance of PartitionProcessor is created anytime an EventProcessor begins receiving messages from a partition.
+ *
+ */
+export class PartitionProcessorBase {
+  private _partitionContext: PartitionContext;
+  private _checkpointManager: CheckpointManager;
+
+  constructor(partitionContext: PartitionContext) {
+    this._checkpointManager = partitionContext.checkpointManager;
+    this._partitionContext = partitionContext;
+  }
+
+  /**
+   * An object that provides information specific to the partition being processed.
```

Comment:
```suggestion
   * An object that provides information specific to the partition being processed. This will have properties like the `partitionId`, `eventHubName` and `consumerGroupName`
```